### ISSUE TITLE
ML-Agents - 'Making a New Learning Environment' problems and unrecognized arguments: config.yaml

### ISSUE THREAD

**Issue by Davidowa** [Unity-Technologies/ml-agents#3442]
**Systems Settings:**
Windows 10
Unity 2019.3.1f1
Barracuda '0.4.0-preview'
Anaconda - Virtual environment "ml-agents" with:
Python 3.7.6
Tensorflow version: 2.0.1
ml-agents: 0.13.1,
ml-agents-envs: 0.13.1
Communicator API: API-13

**Describe the problems:**
1.- I created an New unity project -New Learning Project- where I followed the instructions of  '[Making a New Learning Environment](https://github.com/Unity-Technologies/ml-agents/blob/latest_release/docs/Learning-Environment-Create-New.md)'. I dragged the ML-Agents folder to the project and technically it goes smoothly, but I have a problem with the RollerAcademy script in the editor. It's different from the one in the tutorial. ([See Picture](https://ibb.co/4dm8JqD)) . I'm also posting the scripts I have in the scene. 

2 .- When I pass 
`(ml-agents) D:\>mlagents-learn D:/Unity Projects/ML-agents/ml-agents/config/config.yaml --run-id=RollerBall-1 --train`
in my terminal, as written in the documentation, I get the message: 

`WARNING:tensorflow:From c:\users\dcastill\appdata\local\continuum\anaconda3\envs\ml-agents\lib\site-packages\tensorflow_core\python\compat\v2_compat.py:65: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term`

                        ▄▄▄▓▓▓▓
                   ╓▓▓▓▓▓▓█▓▓▓▓▓
              ,▄▄▄m▀▀▀'  ,▓▓▓▀▓▓▄                           ▓▓▓  ▓▓▌
            ▄▓▓▓▀'      ▄▓▓▀  ▓▓▓      ▄▄     ▄▄ ,▄▄ ▄▄▄▄   ,▄▄ ▄▓▓▌▄ ▄▄▄    ,▄▄
          ▄▓▓▓▀        ▄▓▓▀   ▐▓▓▌     ▓▓▌   ▐▓▓ ▐▓▓▓▀▀▀▓▓▌ ▓▓▓ ▀▓▓▌▀ ^▓▓▌  ╒▓▓▌
        ▄▓▓▓▓▓▄▄▄▄▄▄▄▄▓▓▓      ▓▀      ▓▓▌   ▐▓▓ ▐▓▓    ▓▓▓ ▓▓▓  ▓▓▌   ▐▓▓▄ ▓▓▌
        ▀▓▓▓▓▀▀▀▀▀▀▀▀▀▀▓▓▄     ▓▓      ▓▓▌   ▐▓▓ ▐▓▓    ▓▓▓ ▓▓▓  ▓▓▌    ▐▓▓▐▓▓
          ^█▓▓▓        ▀▓▓▄   ▐▓▓▌     ▓▓▓▓▄▓▓▓▓ ▐▓▓    ▓▓▓ ▓▓▓  ▓▓▓▄    ▓▓▓▓`
            '▀▓▓▓▄      ^▓▓▓  ▓▓▓       └▀▀▀▀ ▀▀ ^▀▀    `▀▀ `▀▀   '▀▀    ▐▓▓▌
               ▀▀▀▀▓▄▄▄   ▓▓▓▓▓▓,                                      ▓▓▓▓▀
                   `▀█▓▓▓▓▓▓▓▓▓▌
                        ¬`▀▀▀█▓

`Version information:
  ml-agents: 0.13.1,
  ml-agents-envs: 0.13.1,
  Communicator API: API-13,
  TensorFlow: 2.0.1
usage: mlagents-learn [-h] [--env ENV_PATH] [--curriculum CURRICULUM_FOLDER]
                      [--sampler SAMPLER_FILE_PATH]
                      [--keep-checkpoints KEEP_CHECKPOINTS] [--lesson LESSON]
                      [--load] [--run-id RUN_ID] [--num-runs NUM_RUNS]
                      [--save-freq SAVE_FREQ] [--seed SEED] [--train]
                      [--base-port BASE_PORT] [--num-envs NUM_ENVS]
                      [--docker-target-name DOCKER_TARGET_NAME]
                      [--no-graphics] [--debug] [--multi-gpu] [--env-args ...]
                      [--cpu] [--version] [--width WIDTH] [--height HEIGHT]
                      [--quality-level QUALITY_LEVEL]
                      [--time-scale TIME_SCALE]
                      [--target-frame-rate TARGET_FRAME_RATE]
                      trainer_config_path
mlagents-learn: error: unrecognized arguments: Projects/ML-agents/ml-agents/config/config.yaml`

and the config.yaml file has

`default:
    trainer: ppo
    batch_size: 10
    beta: 5.0e-3
    buffer_size: 100
    epsilon: 0.2
    hidden_units: 128
    lambd: 0.95
    learning_rate: 3.0e-4
    learning_rate_schedule: linear
    max_steps: 5.0e4
    memory_size: 256
    normalize: false
    num_epoch: 3
    num_layers: 2
    time_horizon: 64
    sequence_length: 64
    summary_freq: 1000
    use_recurrent: false
    vis_encode_type: simple
    reward_signals:
        extrinsic:
            strength: 1.0
            gamma: 0.99`

I don't know what to do. Any help provided will be appreciated.
